### ISSUE TITLE
Add support for CORS configuration in the KafkaBridge resource

### ISSUE THREAD

**Issue by ppatierno** [strimzi/strimzi-kafka-operator#2939]
When the https://github.com/strimzi/strimzi-kafka-bridge/pull/401 will be merged, we have to add support for CORS configuration in the `KafkaBridge` resource in the corresponding `http` section (where today we have just `port`).
Two possible solutions.

```yaml
apiVersion: {KafkaApiVersionPrev}
kind: KafkaBridge
metadata:
  name: my-bridge
spec:
  # ...
  http:
    port: 8080
    cors.allowedOrigins: "some URLs"
    cors.allowedMethods: "GET,POST,PUT,DELETE,OPTIONS,PATCH"
  # ...
```

or

```yaml
apiVersion: {KafkaApiVersionPrev}
kind: KafkaBridge
metadata:
  name: my-bridge
spec:
  # ...
  http:
    port: 8080
    cors:
      allowedOrigins: "some URLs"
      allowedMethods: "GET,POST,PUT,DELETE,OPTIONS,PATCH"
  # ...
```

In both cases, new env vars have to be set for the bridge deployment:

```
KAFKA_BRIDGE_CORS_ENABLED
KAFKA_BRIDGE_CORS_ALLOWED_ORIGINS
KAFKA_BRIDGE_CORS_ALLOWED_METHODS
```

**Comment by stanlyDoge** [comment]
I like the second option.

**Comment by scholzj** [comment]
@ppatierno Is this still a valid issue? Wasn't this done already?

**Comment by ppatierno** [comment]
Fixed by #2994
